web/assets/dark.scss

```diff
@@ -60,11 +62,24 @@ $sirius-bg-color: #29292e;
   .govuk-textarea,
   .govuk-select,
   .govuk-table,
+  .moj-filter,
+  .moj-filter__selected,
   .autocomplete__input {
     background-color: $sirius-bg-color;
     color: govuk-colour("white");
   }
 
+  .moj-pagination__link::before,
+  .moj-pagination__link::after {
+    filter: invert(100%);
+  }
+
+  .moj-pagination__link,
+  #search-results-table th,
+  #search-results-table th button {
+      color: govuk-colour("light-blue");
```

Comment:
the blue colour applied from `aria-sort` is a bit difficult to see in dark mode. `#search-results-table th,` is required as the Address column does not have the `aria-sort=none` attribute